docs_guidelines/how_to/add_code.rst

```diff
@@ -0,0 +1,126 @@
+=========================
+Add code to documentation
+=========================
+
+This guide shows how to include code in your documentation.
+
+There are two ways to have code in your docs:
+
+* As part of the text.
+* In a separate block.
+
+Code as part of the text
+========================
+
+If you want to include a snippet of code inside your text, you only need surround your code with pairs of backquotes, that is,  ````your_code````. So if you want to say that the output of ``2>3`` is ``False``, what you write is:
+
+.. code-block:: text
+
+    the output of ``2>3`` is ``False``
+
+Code blocks
+===========
+
+There are several ways to include code cells in your documentation:
+
+* Executable cells with ``jupyter-execute``.
+* Non-executable cells with ``code-block``.
+* Testable code cells with ``doctest``.
+
+``jupyter-execute``
+-------------------
+
+With ``jupyter-execute`` you can include cells that are executed in a Jupyter kernel and show the output of that code in your documentation. The syntax is:
+
+.. code-block:: text
+
+    .. jupyter-execute::
+
+        your_code
+
+For example, you can run this cell:
+
+.. jupyter-execute::
+
+    from qiskit import QuantumCircuit
+
+    qc = QuantumCircuit(2)
+    qc.h(0)
+    qc.cx(0,1)
+    qc.draw('mpl')
+
+In order to do that, you have to write:
+
+.. code-block:: text
+
+    .. jupyter-execute::
+
+        from qiskit import QuantumCircuit
+
+        qc = QuantumCircuit(2)
+        qc.h(0)
+        qc.cx(0,1)
+        qc.draw('mpl')
+
+.. note::
+
+    For the ``jupyter-execute`` cells to appear you need to add the extension ``jupyter_sphinx`` to the ``docs/conf.py`` of your repository.
+
+
+
+``code-block``
+--------------
+
+If you don't want your code to be executed, you can use ``code-block``, whose syntax is:
```

Comment:
I thought the idea of a how-to is precisely that the user already knows why they want to do things in a determinate way and avoid this kind of discussion. I feel like this could fit better in explanation or a tutorial.

Reply:
Users already know that they want to write code in documentation but not necessarily know what is the better way in what situation. I think it's reasonable to provide options and discuss what options are good for what scenario. I agree that this is close to explanation, but I think this discussion is short enough to include right here.